libraries/engage/core/helpers/i18n.js

```diff
@@ -0,0 +1,58 @@
+import {
+  getTranslator,
+  getPriceFormatter,
+  getDateFormatter,
+  getTimeFormatter,
+  getNumberFormatter,
+} from '@shopgate/pwa-common/helpers/i18n';
+import { logger } from '@shopgate/pwa-core/helpers';
+
+/**
+ * @typedef {Object} I18nHelpers
+ * @property {Function} init - Inits locales.
+ * @property {Function} text - Text translator.
+ * @property {Function} price - Price translator.
+ * @property {Function} date - Date translator.
+ * @property {Function} time - Time translator.
+ * @property {Function} number - Number translator
+ */
+/**
+ * I18n helpers.
+ * @returns {I18nHelpers}
+ */
+const I18n = () => {
+  let didInit = false;
+  /**
+   * Function that throws when called. Used in a default instance of i18n to notify developer
+   * when the module is used without initialization.
+   */
+  const notReadyCb = () => {
```

Comment:
`notReadyCb` is obsolete and just confusing. Could you remove it?

Reply:
I can, I think it's better to throw an error so the developer sees the issue immediately. If it's removed it may be silent.

Reply:
After some conversation off-line I'd remove this safety check.
Just for the record, it will throw generic error when trying to use translations method before `.init` instead.

Reply:
For the record. After more discussions off-line decision is to leave it with warning.